### PR TITLE
REFACTOR LocationPage modeling

### PR DESCRIPTION
**Pull request by muskie9**
Update the overall model pattern to be Page based.

### CODE REVIEW

File: _config/config.yml [hunk 1]
```diff
@@ -7,7 +7,13 @@ Dynamic\Locator\Location:
     - Dynamic\SilverStripeGeocoder\AddressDataExtension
     - SilverStripe\Versioned\Versioned('Stage','Live')
 
-Dynamic\Locator\Locator:
+Dynamic\Locator\Page\LocationPage:
+  extensions:
+    - Dynamic\SilverStripeGeocoder\DistanceDataExtension
+    - Dynamic\SilverStripeGeocoder\AddressDataExtension
+    - SilverStripe\Versioned\Versioned('Stage','Live')
```
Comment (jsirish) [inline _config/config.yml]: do you need Versioned on here?

Comment (muskie9) [reply _config/config.yml]: Sure don't! Good catch!

**Comment by mak001** [comment]
since this will be used in a new major release, are there any objections to implementing #187

**Comment by muskie9** [comment]
@mak001 good idea, let's make sure we get that tackled before releasing the new major.

**Pull request merged**